### PR TITLE
Added a Sentry DSN as secret setting

### PR DESCRIPTION
**Pull request by salmanfarisvp**
I added a Sentry DSN as a secret setting to manage and avoid throwing errors while testing locally and spamming the sentry.

### CODE REVIEW

**Review by sergey-borovkov — CHANGES_REQUESTED** [review]
Please fix the merge properly!

File: edge-apps/tfl-bus-status/static/js/script.js [hunk 1]
```diff
@@ -227,8 +228,11 @@ async function fetchBusData () {
     busArrival.innerHTML = error
     document.querySelector('.bus-list').classList.add('hidden')
   }
-  // Send signal to load the screen once content is ready.
-  screenly.signalReadyForRendering()
+
+  // Send signal to load the screen once content is ready, if the function exists.
+  if (typeof screenly !== 'undefined' && typeof screenly.signalReadyForRendering === 'function') {
```
Comment (sergey-borovkov) [inline edge-apps/tfl-bus-status/static/js/script.js:233]: why are those checks still needed? We discussed we should just not report to sentry.

File: edge-apps/tfl-bus-status/screenly.yml [hunk 2]
```diff
@@ -24,3 +24,16 @@ settings:
     help_text: |
       You can find your bus stop here:
       https://playground.srly.io/edge-apps/helpers/tfl/bus-stop-lookup/
+  sentry_dsn:
+    type: secret
+    title: Sentry Client Key
+    optional: true
+    is_global: true
+    help_text: Sentry Client Key from Sentry SDK.
+  screenly_render_notification:
```
Comment (sergey-borovkov) [inline edge-apps/tfl-bus-status/screenly.yml:33]: ???

File: edge-apps/tfl-bus-status/static/js/script.js (hunk 1 above)
Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/static/js/script.js:233]: I'm sorry, but do you know how we plan to add the sentry?

**Comment by salmanfarisvp** [comment]
> Please fix the merge properly!

Done, please check now. Thanks

**Review by vpetersson — APPROVED** [review]

**Comment by salmanfarisvp** [comment]
@sergey-borovkov, Can you please re-check the previous review?  Thanks.

**Review by sergey-borovkov — APPROVED** [review]

**Pull request merged**